docs/requirements.txt

```diff
@@ -2,4 +2,5 @@ ipython
 scipy
 numpy > 1.9.0
 scikit-learn >= 0.18
+pandas
```

Comment:
it seems ipython is in requirements.txt because of IPython (mock didn't work for some reason)